components/crm/.env.example

```
@@ -53,6 +53,88 @@ ENABLE_TELEMETRY=false
+#
+# SAFETY FLOOR: "token" is rejected when DEPLOYMENT_MODE is saas or byoc, so a
+# production deployment can never authenticate with the dev root token.
+#
+# Per environment: local -> token, dev -> approle, prod (saas/byoc) -> approle.
+KMS_VAULT_AUTH_METHOD=approle
+#
+# VAULT ADDRESS (REQUIRED for envelope mode)
+# KMS_VAULT_ADDR: Vault server address
+#   - Local development: http://midaz-hc-vault:8200 (from infra docker-compose)
+#   - Saas/byoc: https://vault.example.com:8200 (with TLS)
+KMS_VAULT_ADDR=http://midaz-hc-vault:8200
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KMS_VAULT_ADDR key should go before the KMS_VAULT_AUTH_METHOD key

(UnorderedKey)